main/tracker/template.py

```diff
@@ -38,9 +38,8 @@
 license = "GPL-2.0-or-later AND LGPL-2.1-or-later"
 url = "https://gnome.pages.gitlab.gnome.org/tracker"
 source = f"$(GNOME_SITE)/{pkgname}/{pkgver[:-2]}/{pkgname}-{pkgver}.tar.xz"
-sha256 = "52592cfe19baffd16dbe47475be7da750dbd0b6333fd7acb60faa9da5bc40df2"
-# lto fails: Invalid GType function: 'tracker_endpoint_http_get_type'
```

Comment:
Not quite sure what this comment means but the template seems to compile and work fine anyway, and it's been there since the package was added in c51f1fde876a4f5b9296f34ca99e2e88529e1f66 in January 2022, so it may well be outdated.

Reply:
yeah seems outdated